src/common/feedGenerator.ts

```diff
@@ -150,13 +153,30 @@ export const feedToFilters = async (
     )
     .map((setting) => setting.options.type);
 
+  // Split memberships by hide flag
+  const membershipsByHide = memberships.reduce(
+    (
+      acc: { true: string[]; false: string[] },
+      value: { sourceId: SourceMember['sourceId']; hide: boolean },
+    ) => {
+      acc[value.hide ? 'true' : 'false'].push(value.sourceId);
+      return acc;
+    },
+    { true: [], false: [] },
```

Comment:
Can we use something that is not `false | true` since they are boolean keyword tokens in JavaScript so it will look wonky. Maybe `hide | show`

Reply:
Done 👌